src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/Commands.Management.CognitiveServices.csproj

```diff
@@ -0,0 +1,169 @@
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Models\PSCognitiveServicesAccount.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="CognitiveServicesAccount\GetAzureCognitiveServicesAccount.cs" />
+    <Compile Include="CognitiveServicesAccount\GetAzureCognitiveServicesAccountKey.cs" />
+    <Compile Include="CognitiveServicesAccount\GetAzureCognitiveServicesAccountSkus.cs" />
+    <Compile Include="CognitiveServicesAccount\NewAzureCognitiveServicesAccount.cs" />
+    <Compile Include="CognitiveServicesAccount\NewAzureCognitiveServicesAccountKey.cs" />
+    <Compile Include="CognitiveServicesAccount\RemoveAzureCognitiveServicesAccount.cs" />
+    <Compile Include="CognitiveServicesAccount\SetAzureCognitiveServicesAccount.cs" />
+    <Compile Include="CognitiveServicesAccount\CognitiveServicesAccountBaseCmdlet.cs" />
+    <Compile Include="CognitiveServicesAccount\CognitiveServicesManagementClient.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="..\AzureRM.CognitiveServices.psd1">
+      <Link>AzureRM.CognitiveServices.psd1</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="Microsoft.Azure.Commands.Management.CognitiveServices.dll-Help.psd1">
+      <CopyToOutputDirectory>Never</CopyToOutputDirectory>
+    </None>
+    <None Include="Microsoft.Azure.Commands.Management.CognitiveServices.format.ps1xml">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+    <Content Include="Microsoft.Azure.Commands.Management.CognitiveServices.dll-Help.xml">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+      <SubType>Designer</SubType>
+    </Content>
+    <None Include="MSSharedLibKey.snk" />
+    <None Include="packages.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\Common\Commands.Common.Authentication\Commands.Common.Authentication.csproj">
+      <Project>{d3804b64-c0d3-48f8-82ec-1f632f833c9e}</Project>
+      <Name>Commands.Common.Authentication</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\..\Common\Commands.Common\Commands.Common.csproj">
+      <Project>{5ee72c53-1720-4309-b54b-5fb79703195f}</Project>
+      <Name>Commands.Common</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Common\Commands.ResourceManager.Common\Commands.ResourceManager.Common.csproj">
+      <Project>{3819d8a7-c62c-4c47-8ddd-0332d9ce1252}</Project>
+      <Name>Commands.ResourceManager.Common</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Resources\Commands.Resources\Commands.Resources.csproj">
+      <Project>{e1f5201d-6067-430e-b303-4e367652991b}</Project>
+      <Name>Commands.Resources</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Tags\Commands.Tags\Commands.Tags.csproj">
+      <Project>{2493a8f7-1949-4f29-8d53-9d459046c3b8}</Project>
+      <Name>Commands.Tags</Name>
```

Comment:
We can look at moving any needed methods for tag conversion into the common code.  Assembly dependencies between cmdlet projects is no longer allowed (we have our work cut out for us getting rid of the current ones without adding more).